cmd/server/main.go

```diff
@@ -2,20 +2,22 @@ package main
 
 import (
 	"fmt"
+	"log"
+	"net/http"
 
 	"github.com/benchttp/engine/server"
 )
 
-const port = "8080"
+const (
+	port  = "8080"
+	token = "6db67fafc4f5bf965a5a" //nolint:gosec
```

Comment:
Thing is we need to have a mean to go around the cors. 

So the token is needed in this pr.

Are you in favor of not even passing it as a argument from main (where it will be passed from anyway) and in-line it in `server`?

Reply:
We can still return `true` in `CheckOrigin` for now and implement the token logics later (I don't see how it is different from using a token that obtainable from the repo)

Regarding its implementation I'd keep the current one, except we get `token` from an environment variable instead of a hardcoded public value.